scripts/mcptam_install.bash

```diff
@@ -81,35 +97,34 @@ install_dependencies()
 	install_toon
 	install_libcvd
 	install_gvars3
-	rm -rf build_deps
 	cd ..
+	rm -rf build_deps
+
 }
 
 build_mcptam()
 {
-	# setup ros env
-	source $ROS_BASH
-
-	# create catkin workspace
-	mkdir -p $HOME/catkin_ws/src
-	cd $HOME/catkin_ws/src
-	catkin_init_workspace
-	cd -
-
-	# setup catkin workspace
-	cd $HOME/catkin_ws/
-	catkin_make
-	source devel/setup.bash
-	cd -
-
-	# copy mcptam to catkin workspace
-	cd ..
-	cp -R mcptam $HOME/catkin_ws/src
-	cd $HOME/catkin_ws/
-	catkin_make
+
+	if [ "$CI" = true ]; then
+		# grab pull request version to be tested, build done by travis
+		cd ..
+	    cp -R mcptam $CATKIN_WS/src
+	    cd -
+	else
+		# clone mcptam to catkin workspace and build for user
+		cd $CATKIN_WS/src
+		git clone https://github.com/wavelab/mcptam
```

Comment:
Would have the user not already cloned their version of mcptam to there computer? Might it be better to use either a symlink between the current folder at the catkin_ws/src folder or just copy the $PWD to the catkin_ws/src? Right now the user will end up with 2 copies of the mcptam source code on their computer.  Not really a big deal, just a thought though.